libraries/lib-audio-io/AudioIO.cpp

```diff
@@ -1243,7 +1243,7 @@ bool AudioIO::AllocateBuffers(
                ));
                for(auto& buffer : mProcessingBuffers)
                   buffer.reserve(playbackBufferSize);
-               
```

Comment:
@saintmatthieu Please avoid unnecessary formatting changes if you don't change the line itself - it complicates the search through commit history and review process